<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="utf-8">
		<meta http-equiv="X-UA-Compatible" content="IE=edge">
		<meta name="viewport" content="width=device-width, initial-scale=1.0">
		<meta name="description" content="">
		<meta name="author" content="">
		<!-- <link rel="shortcut icon" href="../../docs-assets/ico/favicon.png"> -->

		<title>/apm/.</title>

		<!-- core minified CSS -->
		<link rel="stylesheet" href="./dist/css/bootstrap.min.css">

		<!-- Optional theme(s) -->
		<link rel="stylesheet" href="./dist/css/bootstrap-theme.min.css">
		<link rel="stylesheet" href="./dist/css/jumbotron-narrow.css">
		<link rel="stylesheet" href="./dist/css/font-awesome.min.css">

		<!-- core minified JavaScript -->
		<script src="./dist/js/bootstrap.min.js"></script>

		<!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
		<!--[if lt IE 9]>
			<script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
			<script src="https://oss.maxcdn.com/libs/respond.js/1.3.0/respond.min.js"></script>
		<![endif]-->
	</head>

	<body>

		<div class="container">
			<div class="header">
				<ul class="nav nav-pills pull-right">
					<li class="active"><a href="#">home</a></li>
					<li><a href="#pubs">pulications</a></li>
					<!-- <li><a href="#posters">talks etc</a></li> -->
					<li><a href="./cv.pdf">cv (pdf)</a></li>
				</ul>
				<h3 class="text-muted">Andrew P Morgan, MD PhD</h3>
			</div>

			<div class="row" style="text-align:left;">
				<div class="col-lg-3">
					<img src="me.jpg" alt="photo of self" class="img-circle" style="width:100%;">
				</div>
				<div class="col-lg-9">
					<p class="lead">
					I am a physician-scientist trainee in internal medicine and pediatrics at Duke University Hospital. I am broadly interested in how evolutionary history shapes the distribution of risk alleles in contemporary populations, particularly on the sex chromosomes.
					</p>
				</div>
			</div>

			<div class="row marketing">
				<div class="col-lg-4">
					<h3>Contact</h3>
					<p>Department of Pediatrics<br />
					Duke University Hospital<br />
					4020 North Roxboro St<br />
					Durham NC 27704<br />
					<p>
						<a href="mailto:andrew.parker.morgan@gmail.com"><i class="fa fa-envelope"></i></a>&nbsp;
						<a href="https://github.com/andrewparkermorgan"><i class="fa fa-github"></i></a>&nbsp;
						<a href="https://twitter.com/_apm0"><i class="fa fa-twitter"></i></a>&nbsp;
						<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=morgan+andrew+p%5Bau%5D"><i class="fa fa-book"></i></a>&nbsp;
						<a href="http://orcid.org/0000-0003-1942-4543"><img src="orcid_16x16.gif" /></a>
					<p>
				</div>

				<div class="col-lg-8">
					<h3>Research interests</h3>
					<p>The focus of my thesis was the evolution of the genome of the house mouse (<em>Mus musculus</em>).  In particular I focused on the role of large duplications: these loci are highly variable beween species and contribute disproportionately to genetic variation within populations.  I am interested in the population-genetic forces &mdash; recombination, mutation, selection and drift &mdash; that create and maintain this diversity.
					<p>I was supervised by <a href="https://www.med.unc.edu/genetics/faculty/pardo-manuel">Fernando Pardo-Manuel de Villena,</a> whose lab anchors the <a href="http://csbio.unc.edu/CCstatus/index.py">UNC Systems Genetics</a> research group.</p>
					<p>Besides evolutionary questions, I also worked on the quantiative genetics of complex traits using the <a href="http://csbio.unc.edu/CCstatus/index.py">Collaborative Cross</a> and <a href="http://do.jax.org/">Diversity Outbred</a> mouse populations.</p>
					<p>I am currently a resident in <a href="https://medicine.duke.edu/education-and-training/internal-medicine-residency/duke-program/other-programs/medicine-pediatrics-5">internal medicine and pediatrics</a> at Duke University Hospital.</p>
				</div>
			</div>

			<a name="pubs"></a>
			<div class="row marketing">
				<div class="col-lg-12">
					<h3>Publications and preprints</h3>
					<div class="col-lg-3"><h4>2022<h4></div>
					<div class="col-lg-9">
						<p class="small"><strong>Morgan AP*</strong>, Hughes JJ∗, Didion JP, Jolley WJ, Campbell KJ, Threadgill DW, Bonhomme F, Searle JB, Pardo-Manuel de Villena F (2022) <strong>Population structure and inbreeding in wild house mice (<em>Mus musculus</em>) at different geographic scales.</strong> <em>Heredity</em>, in press.</p>
						<p class="small">preprint: <a href="http://dx.doi.org/10.1101/2022.02.17.478179">10.1101/2022.02.17.478179</a></p>
					</div>
					<div class="col-lg-3"><h4>2021<h4></div>
					<div class="col-lg-9">
						<p class="small">Brazeau NF, Mitchell CL, <strong>Morgan AP</strong>, Deutsch-Feldman M, Watson OJ, Thwai KL, Gelabert P, van Dorp L, Keeler CY, Waltmann A, Emch M, Gartner V, Redelings B, Wray GA, Mwandagalirwa MK, Tshefu AK, Likwela JL, Edwards JK, Verity R, Parr JB, Meshnick SR, Juliano JJ (2021) <strong>The epidemiology of <em>Plasmodium vivax</em> among adults in the Democratic Republic of the Congo</strong>. <em>Nature Communications</em> <strong>12</strong>: 4169.<p>
						<p class="small"><a href="http://dx.doi.org/10.1038/s41467-021-24216-3">10.1038/s41467-021-24216-3</a></p>
						<p class="small">Shrestha B, Shah Z, <strong>Morgan AP</strong>, Saingam P, Chaisatit C, Chaorattanakawee S, Praditpol C, Boonyalai N, Lertsethtakarn P, Wojnarski M, Deutsch-Feldman M, Adams M, Sea D, Chann S, Tyner SD, Lanteri CA, Spring MD, Saunders DL, Smith PL, Lon C, Gosi P, Sok S, Satharath P, Rekol H, Lek D, Vesely BA, Lin JT, Waters NC, Takala-Harrison S (2021) <strong>Distribution and Temporal Dynamics of <em>Plasmodium falciparum</em> Chloroquine Resistance Transporter Mutations Associated With Piperaquine Resistance in Northern Cambodia</strong>. <em>Journal of Infectious Diseases</em> <strong>224</strong>: 1077--1085.<p>
						<p class="small"><a href="http://dx.doi.org/10.1093/infdis/jiab055">10.1093/infdis/jiab055</a></p>
					</div>
					<div class="col-lg-3"><h4>2020<h4></div>
					<div class="col-lg-9">
						<p class="small">Daughety MM, <strong>Morgan AP</strong>, Frost E, Kao C, Hwang J, Tobin R, Patel B, Fuller M, Welsby I, Ortel TL. (2020) <strong>COVID-19 associated coagulopathy: Thrombosis, hemorrhage and mortality rates with an escalated-dose thromboprophylaxis strategy</strong>. <em>Thrombosis Research</em> <strong>196</strong>: 483--485.<p>
						<p class="small"><a href="http://dx.doi.org/10.1016/j.thromres.2020.10.004">10.1016/j.thromres.2020.10.004</a></p>
						<p class="small">Mukaj A, Pialek J, Fotopulosova V, <strong>Morgan AP</strong>, Odenthal-Hesse L, Parvanov ED, Forejt J. (2020) <strong>Prdm9 inter-subspecific interactions in hybrid male sterility of house mouse</strong>. <em>Molecular Biology and Evolution</em> <strong>37</strong>: 3423--3438.<p>
						<p class="small"><a href="http://dx.doi.org/10.1093/molbev/msaa167">10.1093/molbev/msaa167</a></p>
						<p class="small">Verity R, Aydemir O, Brazeau NF, Watson OJ, Hathaway NJ, Mwandagalirwa MK, Marsh PW, Thwai K, Fulton T, Denton M, <strong>Morgan AP</strong>, Parr JB, Tumwebaze PK, Conrad M, Rosenthal PJ, Ishengoma DS, Ngondi J, Gutman J, Mulenga M, Norris DE, Moss WJ, Mensah BA, Myers-Hansen JL, Ghansah A, Tshefu AK, Ghani AC, Meshnick SR, Bailey JA, Juliano JJ (2020) <strong>The impact of antimalarial resistance on the genetic structure of <em>Plasmodium falciparum</em> in the DRC</strong>. <em>Nature Communications</em> <strong>11</strong>: 2107.<p>
						<p class="small"><a href="http://dx.doi.org/10.1038/s41467-020-15779-8">10.1038/s41467-020-15779-8</a></p>
						<p class="small"><strong>Morgan AP*</strong>, Brazeau NF*, Ngasala B, Mhamilawa LE, Denton M, Msellem M, Morris U, Filer DL, Aydemir O, Bailey JA, Parr JB, Martensson A, Bjorkman A, Juliano JJ (2020) <strong>Falciparum malaria from coastal Tanzania and Zanzibar remains highly connected despite effective control efforts on the archipelago</strong>. <em>Malaria J</em> <strong>19</strong>: 47.<p>
						<p class="small"><a href="http://dx.doi.org/10.1186/s12936-020-3137-8">10.1186/s12936-020-3137-8</a></p>
					</div>
					<div class="col-lg-3"><h4>2019<h4></div>
					<div class="col-lg-9">
						<p class="small"><strong>Morgan AP</strong>, Bell TA, Crowley JJ, Pardo-Manuel de Villena F (2019) <strong>Instability of the pseudoautosomal boundary in house mice</strong>. <em>Genetics</em> <strong>212</strong>: 469--487.<p>
						<p class="small"><a href="http://dx.doi.org/10.1534/genetics.119.302232">10.1534/genetics.119.302232</a></p>
					</div>
					<div class="col-lg-3"><h4>2018<h4></div>
					<div class="col-lg-9">
						<p class="small"><strong>Morgan AP*</strong>, Didion JP*, Hughes JJ, Searle JB, Jolley WJ, Campbell KJ, Threadgill DW, Pardo-Manuel de Villena F (2018) <strong>Genetic characterization of invasive house mouse populations on small islands</strong>. <em>bioRxiv</em> 10.1101/332064<p>
						<p class="small"><a href="http://dx.doi.org/10.1101/332064">10.1101/332064</a></p>
					</div>
					<div class="col-lg-3"><h4>2017<h4></div>
					<div class="col-lg-9">
						<p class="small">Rosshart SP, Vassallo BG, Angeletti D, Hutchinson DS, <strong>Morgan AP</strong>, Takeda K, Hickman HD, McCulloch JA, Badger JH, Ajami NJ, Trinchieri G, Pardo-Manuel de Villena F, Yewdell JW, Rehermann B (2017) <strong>Wild mouse gut microbiota promotes host fitness and improves disease resistance</strong>. <em>Cell</em> <strong>171</strong>: 1015--1028.<p>
						<p class="small"><a href="http://dx.doi.org/10.1016/j.cell.2017.09.016">10.1016/j.cell.2017.09.016</a></p>
						<p class="small"><strong>Morgan AP</strong>, Pardo-Manuel de Villena F (2017) <strong>Sequence and structural diversity of mouse Y chromosomes</strong>. <em>Molecular Biology and Evolution</em> <strong>34</strong>: 3186--3204.<p>
						<p class="small"><a href="http://dx.doi.org/10.1093/molbev/msx250">10.1093/molbev/msx250</a></p>
						<p class="small">Makhanova N, <strong>Morgan AP</strong>, Kayashima Y, Makhanov A, Hiller S, Zhilicheva S, Xu L, Pardo-Manuel de Villena F, Maeda N (2017) <strong>Genetic architecture of atherosclerosis dissected by QTL analyses in three F2 intercrosses of apolipoprotein E-null mice on C57BL6/J, DBA/2J and 129S6/SvEvTac backgrounds</strong>. <em>PLoS One</em> <strong>12</strong>: e0182882.<p>
						<p class="small"><a href="http://dx.doi.org/10.1371/journal.pone.0182882">10.1371/journal.pone.0182882</a></p>
						<p class="small">Gralinski LE, Menachery VD, <strong>Morgan AP</strong>, Totura AL, Beall A, Kocher J, Plante J, Harrison-Shostak DC, Schafer A, Pardo-Manuel de Villena F, Ferris MT, Baric RS (2017) <strong>Allelic variation in the Toll-like receptor adaptor protein Ticam2 contributes to SARS-coronavirus pathogenesis in mice</strong>. <em>G3</em> <strong>7</strong>: 1653--1663.<p>
						<p class="small"><a href="http://dx.doi.org/10.1534/g3.117.041434">10.1534/g3.117.041434</a></p>

						<p class="small"><strong>Morgan AP</strong>, Gatti DM, Najarian M, Keane TM, Pack AI, Mott R, Churcill GA, Pardo-Manuel de Villena (2017) <strong>Structural variation shapes the landscape of recombination in mouse.</strong> <em>Genetics</em> <strong>206</strong>: 603--619.</p>
						<p class="small"><a href="https://dx.doi.org/10.1534/genetics.116.197988">10.1534/genetics.116.197988</a>&emsp;&emsp;|&emsp;&emsp;<a href="https://doi.org/10.5281/zenodo.227105"><i class="fa fa-download"></i>&nbsp;data @ Zenodo</a></p>
						<p class="small">Srivastava A*, <strong>Morgan AP*</strong>, Najarian M*, Sarsani VK, Sigmon JS, Shorter JR, Kashfeen A, Giusti-Rodriguez P, Ferris MT, Sullivan PF, Miller DR, Bell TA, McMillan L, Churchill GA, Pardo-Manuel de Villena F (2017) <strong>The genomes of the mouse Collaborative Cross</strong>. <em>Genetics</em> <strong>206</strong>: 537--556.</p>
						<p class="small"><a href="https://dx.doi.org/10.1534/genetics.116.198838">10.1534/genetics.116.198838</a>&emsp;&emsp;|&emsp;&emsp;<a href="http://csbio.unc.edu/CCstatus/CCGenomes/"><i class="fa fa-download"></i>&nbsp;resources @ UNC</a></p>
						<p class="small">Shorter JR, Odet F, Aylor DL, Pan W, Kao CY, Fu CP, <strong>Morgan AP</strong>, Greenstein S, Bell TA, Stevans AM, Feathers RW, Patel S, Cates SE, Shaw GD, Ahmed S, Miller DR, Chesler EJ, McMillian L, O’Brien DA, Pardo-Manuel de Villena F (2017) <strong>Male infertility is responsible for nearly half of the strain extinction observed in the Collaborative Cross</strong>. <em>Genetics</em> <strong>206</strong>: 557--572.</p>
						<p class="small"><a href="https://dx.doi.org/10.1534/genetics.116.199596">10.1534/genetics.116.199596</a></p>
					</div>
					<div class="col-lg-3"><h4>2016<h4></div>
					<div class="col-lg-9">
						<p class="small"><strong>Morgan AP*</strong>, Didion JP*, Doran AG, Holt JM, McMillan L, Keane TM, Pardo-Manuel de Villena F (2016) <strong>Whole genome sequence of two wild-derived <em>Mus musculus domesticus</em> inbred strains, LEWES/EiJ and ZALENDE/EiJ, with different diploid numbers.</strong> <em>G3</em> <strong>6</strong>: 4211--4216</p>
						<p class="small">
						<a href="https://dx.doi.org/10.1534/g3.116.034751">10.1534/g3.116.034751</a>&emsp;&emsp;|&emsp;&emsp;<a href="http://www.ebi.ac.uk/ena/data/view/PRJEB15190"><i class="fa fa-download"></i>&nbsp;data @ ENA: PRJEB15190</a>
						</p>
						<p class="small">Chesler EJ, Gatti DM, <strong>Morgan AP</strong>, Strobel M, Trepanier L, Oberbeck D, McWeeney S, Hitzemann R, Ferris M, McMullan R, Clayshulte A, Bell TA, Pardo-Manuel de Villena F, Churchill GA (2016) <strong>Diversity Outbred Mice at 21: Maintaining allelic variation in the face of selection.</strong> <em>G3</em> <strong>6</strong>: 3893--3902.</p>
						<p class="small"><a href="https://dx.doi.org/10.1534/g3.116.035527">10.1534/g3.116.035527</a></p>
						<p class="small"><strong>Morgan AP</strong>, Holt JM, McMullan RC, Bell TA, Clayshulte AMF, Didion JP, Yadgary L, Thybert D, Odom DT, Flicek P, McMillan L, Pardo-Manuel de Villena F (2016) <strong>The  evolutionary fates of a large segmental duplication in mouse.</strong> <em>Genetics</em> <strong>204</strong>: 267--285.</p>
						<p class="small"><a href="http://dx.doi.org/10.1534/genetics.116.191007">10.1534/genetics.116.191007</a></p>
						<p class="small">Didion JP*, <strong>Morgan AP*</strong>, Yadgary L, Bell TA, McMullan RC, Ortiz de Solorzano L, Britton-Davidian J, Bult CJ, Campbell KJ, Castiglia R, Ching YH, Chunco AJ, Crowley JJ, Chesler EJ, Förster DW, French JE, Gabriel SI, Gatti DM, Garland T Jr, Giagia-Athanasopoulou EB, Giménez MD, Grize SA, Gündüz İ, Holmes A, Hauffe HC, Herman JS, Holt JM, Hua K, Jolley WJ, Lindholm AK, López-Fuster MJ, Mitsainas G, da Luz Mathias M, McMillan L, Ramalhinho MD, Rehermann B, Rosshart SP, Searle JB, Shiao MS, Solano E, Svenson KL, Thomas-Laemont P, Threadgill DW, Ventura J, Weinstock GM, Pomp D, Churchill GA, Pardo-Manuel de Villena F (2016) <strong><em>R2d2</em> drives selfish sweeps in the house mouse.</strong> <em>Molecular Biology and Evolution</em> <strong>33</strong>: 1381--1395.<p/>
						<p class="small"><a href="http://dx.doi.org/10.1093/molbev/msw036">10.1093/molbev/msw036</a></p>
					</div>
					<div class="col-lg-3"><h4>2015<h4></div>
					<div class="col-lg-9">
						<p class="small">
						<strong>Morgan AP*</strong>, Fu CP*, Kao CY, Welsh CE, Didion JP, Yadgary L, Hyacinth L, Ferris MT, Bell TA, Miller DR, Giusti-Rodriguez P, Nonneman RJ, Cook KD, Whitmire JK, Gralinski LE, Keller M, Attie AD, Churchill GA, Petkov P, Sullivan PF, Brennan JR, McMillan L, Pardo-Manuel de Villena F (2015) <strong>The Mouse Universal Genotyping Array: from substrains to subspecies.</strong> <em>G3</em> <strong>6</strong>: 263--297.</p>
						<p class="small">
						<a href="http://dx.doi.org/10.1534/g3.115.022087">10.1534/g3.115.022087</a>&emsp;&emsp;|&emsp;&emsp;<a href="http://datadryad.org/resource/doi:10.5061/dryad.2p7ns"><i class="fa fa-download"></i>&nbsp;data @ Dryad</a>&emsp;&emsp;|&emsp;&emsp;<a href="http://csbio.unc.edu/MUGA"><i class="fa fa-download"></i>&nbsp;resources @ UNC</a>
						</p>
						<p class="small"><strong>Morgan AP.</strong> (2015) <strong><tt>argyle</tt>: an <tt>R</tt> package for analysis of Illumina genotyping arrays.</strong> <em>G3</em> <strong>6</strong>: 281--286.</p>
						<p class="small">
						<a href="http://dx.doi.org/10.1534/g3.115.023739">10.1534/g3.115.023739</a>&emsp;&emsp;|&emsp;&emsp;<a href="https://github.com/andrewparkermorgan/argyle"><i class="fa fa-github"></i>&nbsp;code @ Github</a>
						</p>
						<p class="small"><strong>Morgan AP</strong>, Welsh CE (2015) <strong>Informatics resources for the Collaborative Cross and related mouse populations.</strong> <em>Mammalian Genome</em> <strong>26</strong>: 521--539.</p>
						<p class="small"><a href="http://dx.doi.org/10.1007/s00335-015-9581-z">10.1007/s00335-015-9581-z</a></p>
						<p class="small">Crowley JJ, Zhabotynsky V, Sun W, Huang S, Pakatci IK, Kim Y, Wang JR, <strong>Morgan AP</strong>, Calaway JD, Aylor DL, Yun Z, Bell TA, Buus RJ, Calaway ME, Didion JP, Gooch TJ, Hansen SD, Robinson NN, Shaw GD, Spence JS, Quackenbush CR, Barrick CJ, Nonneman RJ, Kim K, Xenakis J, Xie Y, Valdar W, Lenarcic AB, Wang W, Welsh CE, Fu CP, Zhang Z, Holt J, Guo Z, Threadgill DW, Tarantino LM, Miller DR, Zou F, McMillan L, Sullivan PF, Pardo-Manuel de Villena F (2015) <strong>Analyses of allele-specific gene expression in highly divergent mouse crosses identifies pervasive allelic imbalance.</strong> <em>Nature Genetics</em> <strong>47</strong>: 353--360.</p>
						<p class="small"><a href="http://dx.doi.org/10.1038/ng.3222">10.1038/ng.3222</a></p>
						<p class="small">Didion JP, <strong>Morgan AP</strong>, Clayshulte AM, Mcmullan RC, Yadgary L, Petkov PM, Bell TA, Gatti DM, Crowley JJ, Hua K, Aylor DL, Bai L, Calaway M, Chesler EJ, French JE, Geiger TR, Gooch TJ, Garland T Jr, Harrill AH, Hunter K, McMillan L, Holt M, Miller DR, O'Brien DA, Paigen K, Pan W, Rowe LB, Shaw GD, Simecek P, Sullivan PF, Svenson KL, Weinstock GM, Threadgill DW, Pomp D, Churchill GA, Pardo-Manuel de Villena F (2015) <strong>A multi-megabase copy number gain causes maternal transmission ratio distortion on mouse chromosome 2.</strong> <em>PLoS Genetics</em> <strong>11</strong>: e1004850.</p>
						<p class="small"><a href="http://dx.doi.org/10.1371/journal.pgen.1004850">10.1371/journal.pgen.1004850</a>&emsp;&emsp;|&emsp;&emsp;<a href="http://csbio.unc.edu/r2d2"><i class="fa fa-download"></i>&nbsp;data @ UNC</a></p>
					</div>
					<div class="col-lg-3"><h4>2014<h4></div>
					<div class="col-lg-9">
						<p class="small">Liu EY*, <strong>Morgan AP*</strong>, Chesler EJ, Wang W, Churchill GA, Pardo-Manuel de Villena F (2015) <strong>High-resolution sex-specific linkage maps of the mouse reveal polarized distribution of crossovers in male germline.</strong> <em>Genetics</em> <strong>197</strong>: 91--106.</p>
						<p class="small"><a href="http://dx.doi.org/10.1534/genetics.114.161653">10.1534/genetics.114.161653</a>&emsp;&emsp;|&emsp;&emsp;<a href="http://cgd.jax.org/mousemapconverter/"><i class="fa fa-gear"></i>&nbsp;genetic map converter @ Jax</a></p>
						<p class="small">Rogala AR, <strong>Morgan AP</strong>, Christensen AM, Gooch TJ, Bell TA, Miller DR, Godfrey VL, de Villena FP (2014) <strong>The Collaborative Cross as a resource for modeling human disease: CC011/Unc, a new mouse model for spontaneous colitis.</strong> <em>Mammalian Genome</em> <strong>25</strong>: 95--108.</p>
						<p class="small"><a href="http://dx.doi.org/10.1007/s00335-013-9499-2">10.1007/s00335-013-9499-2</a></p>
						<p class="small"><strong>Morgan AP*</strong>, Crowley JJ*, Nonneman RJ, Quackenbush CR, Miller CN, Ryan AK, Bogue MA, Paredes SH, Yourstone S, Carroll IM, Kawula TH, Bower MA, Sartor RB, Sullivan PF (2014) <strong>The antipsychotic olanzapine interacts with the gut microbiome to cause weight gain in mouse.</strong> <em>PLoS One</em> <strong>9</strong>: e115225.</p>
						<p class="small"><a href="http://dx.doi.org/10.1371/journal.pone.0115225">10.1371/journal.pone.0115225</a></p>
					</div>
				</div>
			</div>

			<!--
			<a name="posters"></a>
			<div class="row marketing">
				<div class="col-lg-12">
					<h3>Talks/posters</h3>
					<div class="col-lg-3"><h4>2016<h4></div>
					<div class="col-lg-9">
						<p class="small"><strong>Sequence and structural diversity of mouse Y chromosomes.</strong> International Congress of Quantiative Genetics 5, Madison, WI, June 2016.</p>
					</div>
					<div class="col-lg-3"><h4>2015<h4></div>
					<div class="col-lg-9">
						<p class="small"><strong>Paradoxical evolution of a large segmental duplication in mouse.</strong> International Mammalian Genome Conference 29, Yokohama, Japan, November 2015.</p>
					</div>
					<div class="col-lg-3"><h4>2014<h4></div>
					<div class="col-lg-9">
						<p class="small"><strong>Natural history of a copy-number variant in mouse.</strong> International Mammalian Genome Conference 28, Bar Harbor, ME, October 2014.</p>
						<p class="small"><strong>A new mouse model for inflammatory bowel disease in the Collaborative Cross.</strong> Complex Traits Consortium 13, Berlin, Germany, May 2014.</p>
					</div>
				</div>
			</div>
			-->

			<div class="footer">
				<p>&copy; APM 2017.  Styled with <a href="http://getbootstrap.com/">Bootstrap</a>.</p>
			</div>

		</div> <!-- /container -->

	</body>
</html>
